src/Kuzzle.ts

```diff
@@ -429,6 +429,8 @@ export class Kuzzle extends KuzzleEventEmitter {
 
   set jwt (encodedJwt) {
     this.auth.authenticationToken = encodedJwt;
+
+    this._loggedIn = encodedJwt ? true : false;
```

Comment:
shouldn't this be a getter checking the existence of `this.auth.authenticationToken` instead of a static member synced by another setter?

Reply:
I can't because in cookieAuth mode there is no JWT property set but that doesn't mean the SDK is not logged out, that why I cannot always check if there is a JWT but I can set a property when people do want to put a JWT by hand.